stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/ImageLoader.kt

```diff
@@ -38,6 +42,29 @@ public object ImageLoader {
             }
     }
 
+    public suspend fun getBitmapUri(
+        context: Context,
+        url: String,
+        transformation: ImageTransformation = ImageTransformation.None
+    ): Uri? = getBitmap(context, url, transformation)?.getLocalBitmapUri(context)
+
+    private fun Bitmap.getLocalBitmapUri(context: Context): Uri? {
+        var bmpUri: Uri? = null
+        try {
+            val file = File(
+                context.getExternalFilesDir(Environment.DIRECTORY_PICTURES),
+                "share_image_" + System.currentTimeMillis() + ".png"
+            )
+            val out = FileOutputStream(file)
+            compress(Bitmap.CompressFormat.PNG, 90, out)
```

Comment:
I'm curious, why png instead of jpeg?

Reply:
JPEG doesn't support alpha channel and quality usually is lower